services/login-sync/bin/arvados-login-sync

```diff
@@ -57,7 +57,9 @@ begin
 
   pwnam = Hash.new()
   logins.reject! do |l|
-    return false if pwnam[l[:username]]
+    if pwnam[l[:username]]
+      false
+    end
```

Comment:
This isn't right.  The remainder of the block needs to be inside an `else` otherwise the `if` statement evaluates false but then it proceeds to evaluate the rest of the block instead of rejecting the user as intended.